### ISSUE TITLE
Changes not reported automatically with MikTex Console

### ISSUE THREAD

**Issue by DavidBitSource** [nvaccess/nvda#14502]
### Steps to reproduce:
MiKTeX Console --> hit check for updates --> focus the Updates tab

### Actual behavior:
NVDA doesn't refresh the Braille display or TTS, so that the number of updates doesn't appear when they do on screen and are not announced neither on the Braille display nor via TTS.
### Expected behavior:
The Braille display and TTS should refresh when there is new content.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2022.4
#### Windows version:
Windows 7 and 10
#### Name and version of other software in use when reproducing the issue:
MiKTeX Console
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
The issue just occures in 2022.4 but not before.
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
Is this an issue only with the MikTex Console?

**Comment by Adriani90** [comment]
@DavidBitSource are you still having this issue with NvDA 2024.3 Beta or 2024.2 on MikTex 24.1 for Windows?

**Comment by Adriani90** [comment]
No updates upon request from the initial reporter since almost 3 years. closing as abandoned. if you are still having this issue, please comment with test results and we can reopen.
